enterprise/server/workflow/workflow.go

```diff
@@ -337,26 +330,38 @@ func (ws *workflowService) checkStartWorkflowPreconditions(ctx context.Context)
 	return nil
 }
 
-func (ws *workflowService) getBazelFlags(ak *tables.APIKey) []string {
+func (ws *workflowService) getBazelFlags(ak *tables.APIKey) ([]string, error) {
 	flags := make([]string, 0)
 	flags = append(flags, "--remote_header=x-buildbuddy-api-key="+ak.Value)
+	if bbURL := ws.env.GetConfigurator().GetAppBuildBuddyURL(); bbURL != "" {
+		u, err := url.Parse(bbURL)
+		if err != nil {
+			return nil, err
+		}
+		u.Path = path.Join(u.Path, "invocation")
+		flags = append(flags, fmt.Sprintf("--bes_results_url=%s/", u))
+	}
 	if eventsAPIURL := ws.env.GetConfigurator().GetAppEventsAPIURL(); eventsAPIURL != "" {
 		flags = append(flags, "--bes_backend="+eventsAPIURL)
 	}
-	return flags
+	return flags, nil
 }
 
 func (ws *workflowService) startWorkflow(webhookID string, r *http.Request) error {
 	ctx := r.Context()
 	if err := ws.checkStartWorkflowPreconditions(ctx); err != nil {
 		return err
 	}
-	webhookData, err := ws.parseRequest(r)
+	// TODO: Support non-GitHub providers.
```

Comment:
you could put this inside of the parseRequest method you removed and switch on r.URL. host or something to call github one appropriately, then have the default case just log "unsupported provider" or something.

Reply:
Good idea, done!

Reply:
After some testing I realized r.URL.Host is empty because this request is from github to our servers. So instead I added a header check to do the classification.